src/bandersnatch/configuration.py

```diff
@@ -83,7 +83,8 @@ def load_configuration(self) -> None:
         config_file = self.default_config_file
         if self.config_file:
             config_file = self.config_file
-        self.config = ConfigParser()
+        self.config = ConfigParser(delimiters=('='))
```

Comment:
Black unfortunately prefers "" ... I just roll with it cause i love auto formatters.